test/helper.ts

```diff
@@ -46,12 +50,13 @@ export async function activate(docUri: vscode.Uri): Promise<any> {
     return activation;
   } catch (e) {
     console.error("Error from activation -> ", e);
+    throw e;
   }
 }
 
 async function reinitializeAnsibleExtension(): Promise<void> {
   await vscode.languages.setTextDocumentLanguage(doc, "ansible");
-  await sleep(2000); //  Wait for server activation (reduced from 20000 to 2000)
+  await sleep(5000); //  Wait for server activation (increased from 2000 to 5000)
```

Comment:
Do we also need this increased wait with the modifications for checking if the server is ready in testYamlWithoutEE?